discovery-dns/src/test/resources/reference.conf

```diff
@@ -0,0 +1,11 @@
+akka {
+
+  discovery {
+    impl = akka.discovery.mock-dns
+
+    # this can be refered to in tests to use the mock discovery implementation
+    mock-dns.class = "akka.discovery.MockDiscovery"
+  }
```

Comment:
probably best to inline this into the tests instead, we have had problems with that reference.conf in test/resources are not always first in classpath